Gruntfile.js

```diff
@@ -85,8 +85,83 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
```

Comment:
:+1:

I still have `Consus-Client-linux.tgz`, `Consus-Client-macos.zip`, and `Consus-Client-win.zip` after cleaning.